react/features/recent-list/components/RecentList.native.js

```diff
@@ -79,6 +82,10 @@ class RecentList extends AbstractRecentList<Props> {
         } = this.props;
         const recentList = toDisplayableList(_recentList, t, _defaultServerURL);
         const slideActions = [ {
+            backgroundColor: ColorPalette.blue,
+            onPress: this._onShowNumbers,
```

Comment:
info

Reply:
Renamed to ```_onShowDialInInfo```. I think that covers the functionality and doesn't depend on the label that we may still change